isobmff-byte-stream-format-respec.html

```diff
@@ -114,6 +136,11 @@
       File Format.
     </section>
 
+    <section id="sotd">
+      <p>The working group maintains <a href="https://github.com/w3c/media-source/issues">a list of all bug reports that the editors have not yet tried to address</a>; there may also be open bugs in the <a href="http://w3.org/brief/Mjcw">previous bug tracker</a>.</p>
+      <p>Implementors should be aware that this specification is not stable. <strong>Implementors who are not taking part in the discussions are likely to find the specification changing out from under them in incompatible ways.</strong> Vendors interested in implementing this specification before it eventually reaches the Candidate Recommendation stage should join the mailing list mentioned below and take part in the discussions.</p>
+    </section>
+
     <section id="introduction">
       <h2>Introduction</h2>
       <p>This specification defines segment formats for implementations that choose to support the ISO Base Media File Format
```

Comment:
Hmm. This file includes a revision history, so I'll update that as part of merging this PR.